doc/manual/citing_aspect.bib

```diff
@@ -1095,3 +1095,17 @@ @Article{geosciences12020050
 ABSTRACT = {The Lau Basin is a back-arc region formed by the subduction of the Pacific plate below the Australian plate. We studied the regional morphology of the back-arc spreading centers of the Northern Lau basin, and we compared it to their relative spreading rates. We obtained a value of 60.2 mm/year along the Northwest Lau Spreading Centers based on magnetic data, improving on the spreading rate literature data. Furthermore, we carried out numerical models including visco-plastic rheologies and prescribed surface velocities, in an upper plate-fixed reference frame. Although our thermal model points to a high temperature only near the Tonga trench, the model of the second invariant of the strain rate shows active deformation in the mantle from the Tonga trench to ~800 km along the overriding plate. This explains the anomalous magmatic production along all the volcanic centers in the Northern Lau Back-Arc Basin.},
 DOI = {10.3390/geosciences12020050}
 }
+
+article{doi:10.1126/sciadv.abg1626,
+author = {Craig O’Neill  and Sonja Aulbach },
```

Comment:
I am pretty sure you need to format "’" differently for pdflatex to work.

Reply:
If you tell pdflatex that you want to use utf8, it actually works just fine.

Regardless, let me merge and fix as a follow-up.